test/custom_task_test.go

```diff
@@ -94,7 +94,7 @@ spec:
     - name: custom-task-ref
       taskRef:
         apiVersion: %s
-        kind: %s
+        customTask: %s
```

Comment:
I think maybe we could just replace the `kind` with `customTask` in the e2e since we don't have kind in the taskref

Reply:
do we also need to keep apiVersion?

Reply:
removed the apiVersion for this and add a check to check the taskref

Reply:
Yeah that is also ok, should I create a new file?